src/runners/JasmineComplianceRunner.ts

```diff
@@ -40,24 +42,37 @@ const defaultJasmineConfigurer = (jasmine) => {
   
     // env.fail
     env.clearReporters(); // Clear default console reporter
-    env.specFilter = () => true
+    // Spec filter on env is deprecated
+    // env.specFilter = () => true
     let reporter = new TSConsoleReporter()
     reporter.setOptions( {showColor : false})
     env.addReporter(reporter);
     //  env.throwOnExpectationFailure(true)
-  
 }
 
+
+
 export class JasmineComplianceRunner {
     jasmine
+    private reportingCompleted: Promise<void>;
+    private basePath;
+    private reportLocation;
     
-    constructor(callback: Callback) {
+    constructor(localExecution?:boolean ) {
```

Comment:
You probably want to change this to take a list of reporters.

You can set a better local path in the junitReporter before you pass it in.